app-crypt/sbsigntools/sbsigntools-0.9.4-r1.ebuild

```diff
@@ -28,6 +28,7 @@ DEPEND="${RDEPEND}
 
 PATCHES=(
```

Comment:
Thanks. I've copied instead of `git mv` and dropped back to ~arch (ekeyword ~all).

I've also added Closes: into the commit message.